package.json

```diff
@@ -254,7 +269,6 @@
                 },
                 {
                     "command": "clangd.switchheadersource",
-                    "when": "resourceLangId == cpp",
```

Comment:
indeed there's nothing limiting this feature to cpp files, in theory all clangd functionality is available for all c-family languages supported by clang.

the problem here is though, if we don't have any when/enablement clause it'll show up on right click on every file and it might become annoying if every plugin did that. So can you just extend the condition to cover c/c++/cuda/objc/objc++ according to https://code.visualstudio.com/docs/languages/identifiers#_known-language-identifiers ?

Reply:
Got it, didn't realize that the context menu appears in all files regardless of the langs listed in `activationEvents`. I changed this to check for the specific c-based languages, and confirmed it's working as expected (shows for objc, doesn't show for py).

Reply:
thanks!